### PR TITLE
[UI] Footer : Add pill with pulsating circle as status link

### LINKED ISSUE #4477: [UI] Update the Status link  to pill with a pulsating indicator
### Current Behavior
Currently our Status link is a plain anchor tag , which does not provide any information about the systems status and is also not much visually appealing 

### Desired Behavior

Change the anchor tag to a pill with the text  `Layer5 Cloud Satus` and a pulsating status indicator 

### Implementation

the status indicator should be :
- green if the systems are up 
- orange if the systems are down 
-
### Acceptance Tests

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Description**
Updates the status link to button with pulsating circle

image
image

This PR fixes #4477

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit cc6b3d4ea7396f3472d5c9b8a7b565eeca3a41d2 at: https://64a9ada7909083284eaddf4c--layer5.netlify.app

**Comment by leecalcote** [comment]
Please use a dark background.

**Review by hirentimbadiya — COMMENTED** [review]
@aabidsofi19 , I think the text is a bit hard to read. It would be great if its color was slightly darker.

**Comment by aabidsofi19** [comment]
image

**Comment by l5io** [comment]
🚀 Preview for commit 30abafbb40d7947827681e9ecb792175d0fad783 at: https://64aa4d984dd25f74e0c77047--layer5.netlify.app

**Comment by samyakshah3008** [comment]
@aabidsofi19  Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would. :)

**Comment by l5io** [comment]
🚀 Preview for commit f163a32d549348bf0ae2873d0b1d5390ff146664 at: https://64aae5f4803f8145a1e0bd31--layer5.netlify.app

**Comment by aabidsofi19** [comment]
@leecalcote latest changes are breaking responsiveness 
image

**Comment by aabidsofi19** [comment]
image

**Comment by leecalcote** [comment]
@aabidsofi19, right. I only made it so far in trying to clean this up. I wonder if you can finish it off, instead. Would you mind doing so?

**Comment by aabidsofi19** [comment]
@leecalcote  yeah sure ...

**Comment by leecalcote** [comment]
Thank you very much.

**Comment by sudhanshutech** [comment]
@aabidsofi19 Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would. :)

**Comment by l5io** [comment]
🚀 Preview for commit 1ebab500a620f10565ab58bfecc900091570bc83 at: https://64aaff03803f815377e0bca5--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5ccf27f46527d755ee916814054cf8baf4fb1446 at: https://64abcdca51861f3bfd3a249e--layer5.netlify.app

**Comment by leecalcote** [comment]
@aabidsofi19 thanks for this. I’m hoping for the Edit this Page to be separate from Privacy and Terms.

Screenshot 2023-07-10 at 2 46 39 PM

**Comment by l5io** [comment]
🚀 Preview for commit c206b4de9245cb728aed4f24e34fcf3d871c1638 at: https://64ad08db2ac5c04f0e2222e3--layer5.netlify.app

**Comment by aabidsofi19** [comment]
@leecalcote  is this good now .

**Comment by leecalcote** [comment]
Yes, it is. I wonder if the pulse would look better if part of it pulsated to a white or a lighter green.

File: src/sections/General/Footer/footer.style.js [hunk 1]
```diff
@@ -278,6 +280,48 @@ const FooterWrapper = styled.section`
...
+		0% {
+			transform: scale(.9);
+		}
+		70% {
+			transform: scale(1);
+			box-shadow: 0 0 0 5px rgba(0,211,169, 0);
+		}
+		100% {
+			transform: scale(.9);
+			box-shadow: 0 0 0 0 rgba(0,211,169, 0);
```
Comment (leecalcote) [inline src/sections/General/Footer/footer.style.js:322]:
```suggestion
			box-shadow: 0 0 0 0 rgba(153,237,220, 0);
```
Trying out a lighter green pulse...

**Review by leecalcote — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 7c2f5b9b508a89da56d6d781258e76dca8fa8971 at: https://64ae853e0cdaa514c19ebf62--layer5.netlify.app

**Pull request merged**